src/citrine/resources/file_link.py

```diff
@@ -648,7 +648,11 @@ def read(self, *, file_link: Union[str, UUID, FileLink]) -> bytes:
         file_link = self._resolve_file_link(file_link)
 
         if self._is_local_url(file_link.url):  # Read the local file
-            path = Path(unquote_plus(url2pathname(urlparse(file_link.url).path)))
+            parsed_url = urlparse(file_link.url)
+            if parsed_url.netloc not in {'', '.', 'localhost'}:
+                raise ValueError("Non-local UNCs (e.g., Windows network paths) are not supported.")
```

Comment:
This is a new restriction, is it not?

Reply:
Formally it's a new restriction, but we would have broken on this case before because we were not considering the netloc value.